test/commands/branch.test.js

```diff
@@ -110,13 +128,92 @@ describe('branch', () => {
         commandArgs: ['$0m3/$7r4ng38r4nChn4m3!'],
         api: [
           () => getProjectByEnv(),
+          () => getEnvironmentListValid(82),
           () => postBranchValid('$0m3/$7r4ng38r4nChn4m3!'),
         ],
+        prompts: [
+          {
+            in: [{
+              name: 'environment',
+              message: 'Select the remote environment you want as origin',
+              type: 'list',
+              choices: ['name1', 'name2'],
+            }],
+            out: {
+              environment: 'name1',
+            },
+          },
+        ],
+
```

Comment:
for the normal case please add the origin in command args, it will reduce drastically the diff